CancerGov/_src/StyleSheets/_homelanding.scss

```diff
@@ -275,20 +276,58 @@ a.learn-more {
 	/* adding padding per #OCECREATIV-783 --sjc */
 }
 
-/* Class used to target homepage only with blue tint on 'Resources for' */
+/* Class previously used to target homepage only with blue tint on 'Resources for' */
 .guide-card--home {
-	div:first-child {
-		background: $lightblue;
+	.card {
+		display: flex;
+		flex-direction: column;
+	}
+	ul {
+		margin-bottom: 0;
+		display: flex;
+		flex-direction: column;
+		flex-grow: 2;
```

Comment:
Flex-grow is a relative value. If all items in a flex container have the same weight, then you should just use flex-grow: 1. It looks like that's the case here.

Reply:
@alifr added the flex-grow:2 to the code.

Reply:
I can't see a reason for it. If you can switch it to 1 without any noticeable effect, you should. It means in the future, nobody will be hunting down the reasoning for it not being the standard 1.

Reply:
updated to flex-grow: 1